libpromises/evalfunction.c

```diff
@@ -2775,6 +2844,22 @@ static FnCallResult FnCallMapData(EvalContext *ctx, ARG_UNUSED const Policy *pol
         return FnFailure();
     }
 
+    if (mapdatamode && jqmode)
+    {
+        // TODO: make this customizable
+        JsonElement *returnjq = ExecJQ(ctx, "/usr/bin/jq", arg_map, container);
+
+        if (NULL == returnjq)
+        {
+            Log(LOG_LEVEL_ERR, "Function '%s' did not get a valid container back from jq after passing variable '%s' and arguments '%s', sorry", fp->name, varname, arg_map);
```

Comment:
This message will be seen too often, I believe, mainly because of missing `jq` executable. So maybe a simpler message, for example "Function %s failed to get output from 'jq' execution".